src/plugins/presentation_util/common/labs.ts

```diff
@@ -50,6 +63,23 @@ export const projects: { [ID in ProjectID]: ProjectConfig & { id: ID } } = {
     }),
     solutions: ['dashboard'],
   },
+  [DASHBOARD_LINKS_PANEL]: {
+    id: DASHBOARD_LINKS_PANEL,
+    isActive: true,
+    isDisplayed: true,
+    environments: ['kibana', 'browser', 'session'],
+    name: i18n.translate('presentationUtil.labs.enableLinksPanelProjectName', {
+      defaultMessage: 'Enable links panel',
+    }),
+    description: i18n.translate('presentationUtil.labs.enableLinksPanelProjectDescription', {
+      defaultMessage:
+        '{technicalPreviewLabel} Enables the links panel for dashboard, which allows dashboard authors to easily link dashboards together.',
+      values: {
+        technicalPreviewLabel: `<em>[${technicalPreviewLabel}]</em>`,
+      },
+    }),
```

Comment:
@amyjtechwriter Some help with the phrasing for this new UI setting would be very appreciated - I really struggled to come up with a succinct description of the links panel 😆 

![image](https://github.com/elastic/kibana/assets/8698078/8bdb73ba-a027-40ff-b6f6-25f59b3d3046)